cypress/tests/data/60-content/VkarbasizaedSubmission.spec.js

```diff
@@ -53,10 +57,151 @@ describe('Data suite tests', function() {
 		cy.get('button').contains('Continue').click();
 		cy.get('button').contains('Continue').click();
 		cy.get('button').contains('Complete').click();
+	});
+
+	it('Schedule for publication', function() {
+		cy.login('dbarnes');
+		// schedule for the publication in the future issue
+		cy.visit('index.php/publicknowledge/submissions');
+		cy.get('button[id="active-button"]').click();
+		cy.contains('View ' + familyName).click({force: true});
+		cy.get('button[id="publication-button"]').click();
+		cy.get('div#publication button:contains("Schedule For Publication")').click();
```

Comment:
Generally, leave the `div` out. The selector is just as specific with `#publication button:contains(...)`. By leaving the `div` out of the selector, we'll be less likely to have to update the test if the HTML is updated slightly.